### ISSUE TITLE
imgHasAlt triggered even if aria-labelledby is used

### ISSUE THREAD

**Issue by robfentress** [ucfopen/UDOIT#98]
The imgHasAlt rule is triggered when an `<img>` tag doesn't have an `alt` attribute set, even if the `<img>` has alternative text using `aria-labelledby`.

![screen shot 2016-02-24 at 4 00 26 pm](https://cloud.githubusercontent.com/assets/9356499/13300796/c2bc6b48-db0f-11e5-9689-41d8f5eaf887.png)
